typescript/README.md

```diff
@@ -0,0 +1,57 @@
+# TypeScript support for Winglang
+
+> TODO
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+`sh
+npm i @winglibs/typescript
+`
+
+## Usage
+
+The `FunctionHandler`, `QueueHandler` and `ApiEndpointHandler` types can be used to define inflight
+handlers from a TypeScript codebase.
+
+For example:
+
+```ts
+// src/index.ts
+exports.handler = async (event, context) => {
+  console.log("Welcome to TypeScript", event);
+  return {
+    status: 200,
+    body: JSON.stringify({ message: "Hello, TypeScript" }),
+  };
+};
```

Comment:
Can we add types to this example?

As a user I'd want some kind of information or disclaimer about what the signature of the function should be and whether it's enforced by the Wing library or not